api/src/middlewares/services/StaffService/__tests__/SESService.test.ts

```diff
@@ -27,13 +27,17 @@ const paymentViewModel = {
 };
 
 test("getEmailBody renders oath email correctly", () => {
-  const emailBody = emailService.getEmailBody({ fields: allOtherFields, payment: paymentViewModel, reference: "1234" }, "submission", "affirmation");
+  const emailBody = emailService.getEmailBody(
+    { fields: allOtherFields, payment: paymentViewModel, reference: "1234", postal: "" },
```

Comment:
could you also write a few other tests to ensure we're treating `postal` correctly? 

e.g. a case for when 
- postal is undefined (i.e not present in the object - as per the type)
- postal is present, but is a falsy value
- postal is present/truthy